inflate_p.h

```diff
@@ -211,3 +211,11 @@ static inline void window_output_flush(PREFIX3(stream) *strm) {
     state->whave += out_bytes;
     state->whave = MIN(state->whave, state->wsize);
 }
+
+/* Behave like chunkcopy, but avoid writing beyond of legal output. */
+static inline uint8_t* chunkcopy_safe(uint8_t *out, uint8_t const *from, unsigned len, uint8_t *safe) {
+    unsigned safelen = (unsigned)((safe - out) + 1);
+    len = MIN(len, safelen);
+    memcpy(out, from, len);
+    return out + len;
```

Comment:
Return value is ignored in some cases, as such `out` needs to be updated.

Reply:
I'm 100% sure modifying the out parameter locally in this function will have no effect whatsoever.  If the calling code relies on a modified out, it is not getting it on HEAD of develop.